.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!-- Thanks for contributing to Cuttle! 🎉 -->
```

Comment:
You can remove this line, it's a comment and doesn't really serve a purpose but I thought it was a nice little message, haha.

Reply:
There's nothing optional about manners ;)